### ISSUE TITLE
[Enhancement] Advertise supported python versions for the fwknop python extension

### ISSUE THREAD

**Issue by Coacher** [mrash/fwknop#167]
Hello.

Currently the fwknop python extension does not specify supported python versions in setup.py or README. This makes using this extension troublesome, since user has to go through the trial-and-error process to determine the required python version. For example, on my machine I am able to use this extension with python 2.7, but not with python 3.3, and I have no clue if this the intended behaviour or not.

Please add the supported Python versions to setup.py. One of the standard ways to do this is through 'classifiers'. See https://docs.python.org/2/distutils/setupscript.html#additional-meta-data Full list: https://pypi.python.org/pypi?%3Aaction=list_classifiers

**Comment by mrash** [comment]
Agreed, fwknop should publish details on Python version dependencies. Thanks for the suggestion.

**Comment by Coacher** [comment]
I've investigated a bit further. On Python 3.4 I have the following error after `fko` module import attempt:

```ImportError:  <long_path_here>/_fko.cpython-34.so: undefined symbol: Py_InitModule```

This error is due to changes in Python 3 module initialization.
See https://docs.python.org/3/howto/cporting.html#module-initialization-and-state
And http://python3porting.com/cextensions.html#module-initialization

Therefore `fko` python module does not have proper Python 3 support atm. Please adjust classifiers accordingly.

**Comment by anthraxx** [comment]
just an idea, but how about fixing python3 support instead of documenting that its only functional for python2? :stuck_out_tongue_winking_eye:

**Comment by hlein** [comment]
Python3 support would be helpful since Python2 is deprecated, and distributions are starting to remove it.

I'm a terribad python programmer, or I would volunteer to help.

**Comment by exx8** [comment]
As python 2 has been decrepated, I suggest moving py script altogether to py 3, I might be able to offer some help  within half year from now. Will a PR porting to python 3 will be accepted?
